composer.json

```diff
@@ -15,12 +15,18 @@
         "adodb/adodb-php": "5.20.2",
         "phpmailer/phpmailer": "5.2.16",
         "rospdf/pdf-php": "0.12.22",
-        "phpseclib/phpseclib": "2.0.2"
-        
+        "phpseclib/phpseclib": "2.0.2",
+        "zendframework/zendframework": "2.4.9",
+        "dino/dompdf-module": "dev-master",
+        "doctrine/couchdb": "@dev",
```

Comment:
Shouldn't the above 2 be set to a specific version?
(preferably the version that was previously used)